ambari-server/src/main/resources/stacks/HDP/3.0

```diff
@@ -0,0 +1 @@
+/Users/jhurley/src/hwx/hdp_ambari_definitions/src/main/resources/stacks/HDP/3.0
```

Comment:
This seems accidental... yes?

Reply:
Totally ... part of testing involved getting the old packs on my box.